gluon:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50  

  # Position Controllers ---------------------------------------
  left_joint1_position_controller:
    type: effort_controllers/JointPositionController
    joint: left_axis_joint_1
    pid: {p: 50.0, i: 0.01, d: 1.0}
  
  left_joint2_position_controller:
    type: effort_controllers/JointPositionController
    joint: left_axis_joint_2
    pid: {p: 40.0, i: 0.01, d: 1.0}
    
  left_joint3_position_controller:
    type: effort_controllers/JointPositionController
    joint: left_axis_joint_3
    pid: {p: 10.0, i: 0.01, d: 0.1}

  left_joint4_position_controller:
    type: effort_controllers/JointPositionController
    joint: left_axis_joint_4
    pid: {p: 4.0, i: 0.01, d: 0.1}

  left_joint5_position_controller:
    type: effort_controllers/JointPositionController
    joint: left_axis_joint_5
    pid: {p: 2, i: 0.05, d: 0.05}

  left_joint6_position_controller:
    type: effort_controllers/JointPositionController
    joint: left_axis_joint_6
    pid: {p: 1, i: 0.05, d: 0.05}

  right_joint1_position_controller:
    type: effort_controllers/JointPositionController
    joint: right_axis_joint_1
    pid: {p: 50.0, i: 0.01, d: 1.0}
  
  right_joint2_position_controller:
    type: effort_controllers/JointPositionController
    joint: right_axis_joint_2
    pid: {p: 40.0, i: 0.01, d: 1.0}
    
  right_joint3_position_controller:
    type: effort_controllers/JointPositionController
    joint: right_axis_joint_3
    pid: {p: 10.0, i: 0.01, d: 0.1}

  right_joint4_position_controller:
    type: effort_controllers/JointPositionController
    joint: right_axis_joint_4
    pid: {p: 4.0, i: 0.01, d: 0.1}

  right_joint5_position_controller:
    type: effort_controllers/JointPositionController
    joint: right_axis_joint_5
    pid: {p: 2, i: 0.05, d: 0.05}

  right_joint6_position_controller:
    type: effort_controllers/JointPositionController
    joint: right_axis_joint_6
    pid: {p: 1, i: 0.05, d: 0.05}

  cloud_platform_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: cloud_platform_joint
    pid: {p: 3.0, i: 0.01, d: 1.0}

gluon2:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50  

  # Position Controllers ---------------------------------------
  left_joint1_position_controller:
    type: effort_controllers/JointPositionController
    joint: left_axis_joint_1
    pid: {p: 300.0, i: 0.01, d: 1.0}
  
  left_joint2_position_controller:
    type: effort_controllers/JointPositionController
    joint: left_axis_joint_2
    pid: {p: 80.0, i: 0.01, d: 1.0}
    
  left_joint3_position_controller:
    type: effort_controllers/JointPositionController
    joint: left_axis_joint_3
    pid: {p: 40.0, i: 0.01, d: 0.1}

  left_joint4_position_controller:
    type: effort_controllers/JointPositionController
    joint: left_axis_joint_4
    pid: {p: 10.0, i: 0.01, d: 0.1}

  left_joint5_position_controller:
    type: effort_controllers/JointPositionController
    joint: left_axis_joint_5
    pid: {p: 2, i: 0.05, d: 0.05}

  left_joint6_position_controller:
    type: effort_controllers/JointPositionController
    joint: left_axis_joint_6
    pid: {p: 1, i: 0.05, d: 0.05}

  right_joint1_position_controller:
    type: effort_controllers/JointPositionController
    joint: right_axis_joint_1
    pid: {p: 300.0, i: 0.01, d: 1.0}
  
  right_joint2_position_controller:
    type: effort_controllers/JointPositionController
    joint: right_axis_joint_2
    pid: {p: 80.0, i: 0.01, d: 1.0}
    
  right_joint3_position_controller:
    type: effort_controllers/JointPositionController
    joint: right_axis_joint_3
    pid: {p: 40.0, i: 0.01, d: 0.1}

  right_joint4_position_controller:
    type: effort_controllers/JointPositionController
    joint: right_axis_joint_4
    pid: {p: 20.0, i: 0.01, d: 0.1}

  right_joint5_position_controller:
    type: effort_controllers/JointPositionController
    joint: right_axis_joint_5
    pid: {p: 4, i: 0.05, d: 0.05}

  right_joint6_position_controller:
    type: effort_controllers/JointPositionController
    joint: right_axis_joint_6
    pid: {p: 2, i: 0.05, d: 0.05}

  cloud_platform_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: cloud_platform_joint
    pid: {p: 50.0, i: 0.01, d: 1.0}